src/libxtp/espfit.cc

```diff
@@ -112,7 +116,10 @@ void Espfit::Fit2Density(std::vector< QMAtom* >& _atomlist, ub::matrix<double> &
 ub::vector<double> Espfit::EvalNuclearPotential(std::vector< QMAtom* >& _atoms, Grid _grid) {
     ub::vector<double> _NucPatGrid = ub::zero_vector<double>(_grid.getsize());
 
+<<<<<<< HEAD
    
+=======
+>>>>>>> fc32ec7d18c290b005462342e604b450817c3563
```

Comment:
Same!

Reply:
Yes, yes. You are highlighting my inexperience with git. :)